tests/route/test_static_route.py

```diff
@@ -157,7 +158,7 @@ def _check_routes():
                 return False
         return True
 
-    assert(wait_until(60, 15, 0, _check_routes))
+    assert wait_until(60, 15, 0, _check_routes)
```

Comment:
is this an unrelated change?

Reply:
Yes. contentcheck was complaning about this line, so thought of fixing this.
```flake8...................................................................Failed
- hook id: flake8
- exit code: 1

tests/route/test_static_route.py:161:11: E275 missing whitespace after keyword```

Reply:
Basically, `flake8` hook was failing during `git commit` and thus needed to make this change.